### ISSUE TITLE
xbrz.cpp.o fails to build on Ubuntu?

### ISSUE THREAD

**Issue by HimbeersaftLP** [Tinob/Ishiiruka#124]
I think I am too stupid for this. Sorry for wasting your time.

Output of make command: https://gist.github.com/HimbeersaftLP/2c8d9da2b2b9def3a48d3660f4d1fa47

My System:
OS: Ubuntu 18.04.1 LTS
running in VirtualBox

**Comment by billywade** [comment]
Hey, don't worry about anything. Everyone starts somewhere.

Is there any particular reason you aren't building globally?

**Comment by HimbeersaftLP** [comment]
I was building it to use it later on another system that wasn't available at the time.

**Comment by billywade** [comment]
Yeah, so if you don't run ```sudo make install``` and run the build somewhere other than ```/tmp``` you can keep it around indefinitely and then copy the build directory whenever. If you would try the build without the ```-DLINUX_LOCAL_DEV=true``` flag and report back, I might be able to help more from there.

**Comment by HimbeersaftLP** [comment]
Building without the `DLINUX_LOCAL_DEV=true` flag appears to work just fine.
But when I start the binary from the Binaries folder (I haven't run `make install`) it's missing all resources (icons etc.).

**Comment by HimbeersaftLP** [comment]
This is what the window looks like when I run it from the `Binaries` folder:
![image](https://user-images.githubusercontent.com/16354747/48302948-fb628300-e503-11e8-83c6-8987ea2288e2.png)
This is the console output:
```himbeer@himbeer-VirtualBox:~/Schreibtisch/Ishiiruka/build/Binaries$ ./ishiiruka
Gtk-Message: 15:14:58.520: Failed to load module "canberra-gtk-module"```

**Comment by billywade** [comment]
The Canberra thing can be ignored, it's a GTK audio thing that more or less just irritates users until it's installed. 

It looks like what's happening is that Ishiiruka is expecting asset files to exist in places they would be if they were fully installed, but instead is breaking because they aren't. It should be trying to search locally before breaking. 

Can you try compiling and running the WXWidgets version of Dolphin and posting the output? This might be an upstream issue, so I want to make sure the issue goes to the right place.

**Comment by HimbeersaftLP** [comment]
> Can you try compiling and running the WXWidgets version of Dolphin

How would I do that? I'm a noob.
